CODE_OF_CONDUCT.md

```diff
@@ -168,41 +203,43 @@ Our ability to address any code of conduct breaches in a timely and effective ma
 - Names and contact info, if possible, of **anyone else who witnessed** or was involved in this incident. (Did anyone else observe the incident?)
 - **Any other relevant information** you believe we should have
 
-At conda Organization Events: Event staff will attempt to gather and write down the above information from anyone making a verbal report in-person at an event. Recording the details in writing is exceedingly important in order for us to effectively respond to reports. If event staff write down a report taken verbally, then the person making the report will be asked to review the written report for accuracy.
+## Anonymous Reporting
+
+The reporting form supports anonymous incident reporting.  Anonymous reporting works best when the behavior happened in a public space and was witnessed by many.  If an incident is reported anonymously and was not witnessed by others, than the committee may be limited in what actions it can take and what it can report to the larger community.  Nevertheless, the CoC Committee is still interested in receiving these reports.  They are helpful when determining what we need to address as a community, and when looking for evidence of repeated behavior.
 
-**If urgent action is needed regarding an incident at an in-person event, we strongly encourage you to reach out to the local event staff for immediate assistance.**
+## Conflicts of Interest
 
-### Enforcement: What Happens After a Report is Filed?
+Committee members are expected to recuse themselves in they have a conflict of interest. In addition, the CoC Committee can [vote](#voting-and-decision-making) to remove a committee member from a case, if the committee feels that the member has a conflict of interest.  This [vote](#voting-and-decision-making) requires a simple majority.
```

Comment:
This may need to be stronger, forcing the recusal of any CoC Committee member named in a case (as reporter or accused). 

I am making this comment solely in my personal capacity and am not conveying any rights to any intellectual property of any third parties.

Reply:
@CJ-Wright Absolutely, and I think this is uncontroversial.  Look for an update in the next PR.

Reply:
Done, @CJ-Wright does that work?